src/components/archive.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Is this still necessary? I know you no longer need to do this with CRA.

Reply:
this is Gatsby, so yes, we need the import